src/module-elasticsuite-catalog/view/adminhtml/templates/catalog/product/attribute/js.phtml

```diff
@@ -14,354 +14,354 @@
-        'Magento_Ui/js/modal/alert',
-        'Magento_Ui/js/modal/prompt',
-        "collapsable",
-        "prototype"
-    ], function(jQuery, alert, prompt){
-
-        function toggleApplyVisibility(select) {
-            if ($(select).value == 1) {
-                $(select).next('select').removeClassName('no-display');
-                $(select).next('select').removeClassName('ignore-validate');
-
-            } else {
-                $(select).next('select').addClassName('no-display');
-                $(select).next('select').addClassName('ignore-validate');
-                var options = $(select).next('select').options;
-                for( var i=0; i < options.length; i++) {
-                    options[i].selected = false;
-                }
-            }
+require([
+    "jquery",
+    'Magento_Ui/js/modal/alert',
+    'Magento_Ui/js/modal/prompt',
+    'uiRegistry',
+    "collapsable",
+    "prototype"
+], function(jQuery, alert, prompt, registry){
+
+function toggleApplyVisibility(select) {
+    if ($(select).value == 1) {
+        $(select).next('select').removeClassName('no-display');
+        $(select).next('select').removeClassName('ignore-validate');
+
+    } else {
+        $(select).next('select').addClassName('no-display');
+        $(select).next('select').addClassName('ignore-validate');
+        var options = $(select).next('select').options;
+        for( var i=0; i < options.length; i++) {
+            options[i].selected = false;
+        }
+    }
+}
+function getFrontTab() {
+    if ($('product_attribute_tabs_front')) {
+        return $('product_attribute_tabs_front').up('li');
+    } else {
+        return $('front_fieldset-wrapper');
+    }
+}
+
+function checkOptionsPanelVisibility(){
+    if($('manage-options-panel')){
+        var panelId = 'manage-options-panel',
+            panel = $(panelId),
+            panelFieldSet = panel.up('.fieldset'),
+            activePanelClass = 'selected-type-options';
+
+        if($('frontend_input') && ($('frontend_input').value=='select' || $('frontend_input').value=='multiselect')){
+            panelFieldSet.show();
+            jQuery(panel).addClass(activePanelClass);
+            registry.get(panelId, function () {
+                jQuery('#' + panelId).trigger('render');
+            });
+        }
+        else {
+            panelFieldSet.hide();
+            jQuery(panel).removeClass(activePanelClass);
         }
-        function getFrontTab() {
-            if ($('product_attribute_tabs_front')) {
-                return $('product_attribute_tabs_front').up('li');
-            } else {
-                return $('front_fieldset-wrapper');
+    }
+}
+
+function bindAttributeInputType()
+{
+    checkOptionsPanelVisibility();
+    switchDefaultValueField();
+    if($('frontend_input') && ($('frontend_input').value=='boolean' || $('frontend_input').value=='select' || $('frontend_input').value=='multiselect' || $('frontend_input').value=='price')){
```

Comment:
The (added by our module) condition for 'text' is missing.

Reply:
Indeed, what's the point to copy upstream otherwise :man_facepalming: 
Sorry, I was in a hurry on that day. Good catch.